pandas/core/base.py

```diff
@@ -842,6 +842,13 @@ def _map_values(self, mapper, na_action=None):
                 )
 
         if isinstance(mapper, ABCSeries):
+            # na_action must be None or "ignore"
+            if na_action not in (None, "ignore"):
```

Comment:
A test is needed to exercise this `if` branch

Reply:
Added a test, thank you!